src/util/encryption.ts

```diff
@@ -0,0 +1,8 @@
+import { BinaryLike, createHmac } from "crypto";
+import { envVars } from "../config/env";
+
+export const createHashWithSharedSecret = (data: BinaryLike, secret = envVars.GLOBAL_HASH_SECRET): string => {
```

Comment:
Wondering why it is "shared"? :)

Reply:
you mean just have it as createHash instead of createHashWithSharedSecret or createHashWithGlobalSecret?

Reply:
Well, since you are passing the secret as a parameter, I think it should be just "createHash". TBH I liked more when it was createHashWithGlobalSecret and without the parameter 🤷 . Anyway, pretty sure this is a "nit" 😃

Reply:
sigh.. your right, ready to re-re-re-reapprove lolol
its either createHashWithSharedSecret without the default value or
createHash with the default value. combination is just messy.

1 moment.